ggml/src/ggml-cuda/mmq.cuh

```diff
@@ -1376,7 +1376,7 @@ template <int mmq_y, int nwarps, bool need_check> static __device__ __forceinlin
         const half2 dm = bxi->dm * make_half2(1.0f, -1.0f);
 
 #pragma unroll
-        for (int l = 0; l < sizeof(int); ++l) {
+        for (size_t l = 0; l < sizeof(int); ++l) {
```

Comment:
These loop variable are likely better to remain as `int` to reduce registers usage. But I'll defer to Johannes for that, as I am not 100% it makes a difference with CUDA.